src/main/java/com/j256/ormlite/table/TableUtils.java

```diff
@@ -100,16 +100,6 @@ public static int createTable(Dao<?, ?> dao) throws SQLException {
 		return doCreateTable(dao, true);
 	}
 
-
```

Comment:
Is there a reason you did this in this check-in or was this a mistake?

Reply:
Everything else looks good.

Reply:
You asked me to do so (probably it was misunderstanding)
https://github.com/j256/ormlite-core/pull/173#discussion_r389028794

Reverted

Reply:
Sorry, I thought there was a method that was removed.  I'm not seeing that diff now.

Reply:
Ok. Should I do something else or it can be merged now?